.circleci/config.yml

```diff
@@ -738,7 +787,20 @@ jobs:
     resource_class: medium
     <<: *environment
     steps:
-      - setup_remote_docker
+      - when:
+          condition:
+            or:
+              - << pipeline.parameters.instance_tests >>
+              - << pipeline.parameters.contrib_branch >>
+              - << pipeline.parameters.demisto_sdk_nightly >>
+              - << pipeline.parameters.nightly >>
```

Comment:
I think 6.0 should run in nightly. Not sure about non_ami_run. @yaakovi fyi

Reply:
My bad, you're right

Reply:
non_ami_run runs in 5_0 currently, fixing it too